media/css/firefox/developer-2017.scss

```diff
@@ -0,0 +1,478 @@
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import
+    '../pebbles/includes/lib',
+    '../pebbles/components/newsletter';
+
+$color-devblue: #161937;
+$color-devblue-alt: #1d2142;
+$color-devteal: #00c8d9;
+
+
+.firefox-developer {
+    #outer-wrapper {
+        background-color: $color-devblue;
+    }
+
+    #masthead {
+        @include open-sans;
+        background: $color-devblue;
+        color: #fff;
+
+        a:link,
+        a:visited {
+            background: transparent;
+            color: #fff;
+        }
+
+        .masthead-nav-main {
+            text-transform: none;
+
+            .toggle {
+                background-image: url('/media/img/pebbles/icon-menu-light.svg');
+            }
+
+            .nav-main-menu {
+                background: $color-devblue;
+
+                li {
+                    border-bottom-color: $color-devblue-alt;
+                }
+
+                a:hover,
+                a:focus,
+                a:active {
+                    border-right-color: $color-devteal;
+                }
+
+                @media #{$mq-tablet} {
+                    background: transparent;
+
+                    li {
+                        border-bottom: 0;
+                    }
+
+                    a:hover,
+                    a:focus,
+                    a:active {
+                        border-right: 0;
+                    }
+                }
+            }
+        }
+    }
+
+    // Conflicting download button in global nav is confusing.
+    // Hide it for now pending a better solution.
+    #global-nav-download-firefox,
+    #nav-download-firefox {
+        display: none;
+    }
+
+    main {
+        @include open-sans;
+        color: #fff;
+
+        a:link,
+        a:visited {
+            color: $color-devteal;
+
+            &:hover,
+            &:focus,
+            &:active {
+                color: #fff;
+            }
+        }
+
+        .content {
+            max-width: $width-desktop;
+            padding: 40px 20px;
+        }
+    }
+
+    .section-title {
+        @include font-size-level3;
+        margin: 20px auto 40px;
+        max-width: $width-tablet;
+        text-align: center;
+    }
+
+    .hero-image {
+        display: block;
+        margin: 0 auto 40px;
+
+        @media #{$mq-desktop} {
+            max-width: 900px;
+        }
+    }
+
+    .section-cta {
+        text-align: center;
+
+        a.button-hollow {
+            @include font-size(18px);
+            border-color: $color-devteal;
+            border-radius: 0;
+            color: $color-devteal;
+
+            &:hover,
+            &:focus,
+            &:active {
+                background-color: transparentize($color-devteal, .9);
+                border-color: lighten($color-devteal, 10%);
+                color: lighten($color-devteal, 10%);
+            }
+        }
+    }
+
+    // Download button
+    .download-button {
```

Comment:
If this is another new download button style, would it be worth consolidating into buttons.scss?